config/samples/nemo/latest/apps_v1alpha1_nemodatastore.yaml

```diff
@@ -31,7 +31,7 @@ spec:
   pvc:
     name: "pvc-shared-data"
     create: true
-    storageClass: "local-path"
+    storageClass: ""
```

Comment:
since the storageClass is optional, maybe we can leave it out of the CR.
I worry that this might confuse the users into believing the resulted PVC will have an empty string for the `.spec.storageClassName` field.

Reply:
or maybe use a placeholder for storageClass
```
storageClass: <my-storage-class>
```

Reply:
In the docs, thats how they are, but the samples we have are configured with defaults from ansible. There are cases for e.g. TKGS where a default SC is already there and causing issues with the default sample.